src/syscheckd/db/src/db.cpp

```diff
@@ -61,6 +61,7 @@ void fim_db_init(int storage,
         FIMDBHelper::initDB<FIMDB>(sync_interval, file_limit, value_limit, sync_callback, log_callback, dbsyncHandler,
                                    rsyncHandler);
 #endif
+        free(createQuery);
```

Comment:
Memory management should be all in c ++

Reply:
Done 
https://github.com/wazuh/wazuh/pull/11383/commits/e55d190fec52fda406855238061869c3eb935bcc